pkgs/os-specific/linux/cramfsprogs/include-sysmacros.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/mkcramfs.c b/mkcramfs.c
+index a2ef018959d..bec83c112d1 100644
+--- a/mkcramfs.c
++++ b/mkcramfs.c
+@@ -22,6 +22,7 @@
+  * If you change the disk format of cramfs, please update fs/cramfs/README.
+  */
+ 
++#include <sys/sysmacros.h>
```

Comment:
should this be upstreamed? doesn't seem NixOS specific

Reply:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=800316 states that:
> cramfs is unmaintained upstream, and in the kernel is deprecated in favour of squashfs

Which https://tracker.debian.org/pkg/cramfs confirms:
> This package is not in any development repository.

Reply:
:(, do you mind adding a comment then? that's unfortunate